docs/customization/docker.rst

```diff
@@ -18,7 +18,7 @@ You will need to install Docker and `docker-compose <https://docs.docker.com/com
 
 .. note::
 
-   Our configuration exposes PostgreSQL, Redis and Elasticsearch ports. If you have problems running this docker file because of port conflicts, you can remove ``ports`` section from ``docker-compose.yml``.
+   Our configuration uses a ``docker-compose.override.yml`` that exposes PostgreSQL, Redis and Elasticsearch ports. If you don't want to expose the ports you can tell Docker Compose to do not include the additional configurations (ports) in the ``docker-compose.override.yml`` file by specifying ``docker-compose.yml`` with the ``-f`` option, as in ``docker-compose -f docker-compose.yml up -d``.
```

Comment:
> you can tell Docker Compose **to do not** include the additional configurations

I think you should change it to "not to include".

Reply:
Thanks! Done.